src/apps/watchfaces/OswAppWatchfaceMix.cpp

```diff
@@ -93,23 +93,37 @@ void OswAppWatchfaceMix::digitalWatchDisplay() {
 }
 
 void OswAppWatchfaceMix::setup() {
-
+    uint32_t nowDay, dayOfMonth = 0;
+    OswHal::getInstance()->getLocalDate(&dayOfMonth, &nowDay);
+    pos = nowDay;
 }
-
 void OswAppWatchfaceMix::loop() {
     OswHal* hal = OswHal::getInstance();
-    if (hal->btnHasGoneDown(BUTTON_3)) {
-        hal->increaseBrightness(25);
+    if (hal->btnIsDown(BUTTON_3) && hal->btnHasGoneDown(BUTTON_2)) {
+        buttonMode = (buttonMode + 1) % 2;
     }
-    if (hal->btnHasGoneDown(BUTTON_2)) {
-        hal->decreaseBrightness(25);
+    ui->watchfaceBrightnessControlMode = buttonMode;
+
+    switch (buttonMode) {
```

Comment:
This would require every app to also have this `switch`-statement in it by itself. I think this will make future extension pretty hard! How about you create a static method inside the watchface / UI class, which handles the buttons and expects e.g. a callback for an additional button mode called "app-specific" or "other" or "custom". If the user selects this mode (own icon?), the callback is just being called whenever the static method is being executed from the app.